.github/workflows/ci.yaml

```diff
@@ -21,6 +21,9 @@ jobs:
                     - '7.3'
                     - '7.4'
                 composer_options: [ "" ]
+                include:
+                    -   php: '8.0'
```

Comment:
We might consider switching to 8.1 since it was released today.

Reply:
On kernel/core we can safely add both, because open source projects have unlimited GH Action minutes.